### DISCUSSION TITLE
Can i in "Almost maximized" change the default size of windows, or decrease the border?

### DISCUSSION THREAD

**Discussion by ghost** [rxhanson/Rectangle#1112] [Q&A]
Can i in "Almost maximized" change the default size of windows, or decrease the border?

**Comment by gothic-bum** [comment]
You can set the size of "Almost Maximize" to a percentage:  
https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#almost-maximize

**Reply by 9mm** [reply]
WOW!!!!! Holy crap, god bless. I was so depressed because this window manager is SO GOOD but i can't do the following

- a small margin on bottom so it doesnt touch the dock
- a better ratio for the center (i want it to also resize height)

using that link it looks like i can actually do both those...

my question is, do those also work on Rectangle Pro? @gothic-bum  @rxhanson

**Reply by rxhanson** [reply]
@9mm Yes, those terminal commands also work in Rectangle Pro, just replace `com.knollsoft.Rectangle` with `com.knollsoft.Hookshot` (Pro's original name). There's a custom shortcut builder in Pro, so you don't have to use a terminal command to get a centered window with a different ratio. And I'm planning on adding screen edge gaps into the UI of Pro, hopefully in one of the next releases.

**Reply by 9mm** [reply]
Is there also a command to remove the gap between windows when doing half left and half right?

**Reply by rxhanson** [reply]
Not currently. The only workaround I can think of at the moment would be to build out separate custom shortcuts in Rectangle Pro that use absolute positioning and would ignore the gap setting.

**Reply by 9mm** [reply]
I see what you mean about absolute positioning. Even when I create a custom shortcut that is 0.5 / 1.0, right-docked, the footprint looks correct but the actual window does not match the footprint (the gap is still there)

Is there an easy way I can see the dimensions of my screen?

**Reply by 9mm** [reply]
nevermind i just looked in about this mac and its there

**Reply by 9mm** [reply]
Ok last question -- is there any way i can clear any sort of keyboard cache or something? I think i have created a bug where I cannot assign `c+s+d+Right arrow` for my hotkey. I have tried to do/undo it so many times it started glitching out. That exact keyboard shortcut doesnt exist elsewhere in the app (it did previously and I erased). Now when I push it, theres no error sound, it just will not 'take' it

**Reply by 9mm** [reply]
Whats weird is that AS I push that hotkey while creating custom shortcuts it DOES move the rectangle window... yet when i go to any other window and do the same hotkey it does not move. This is what lead me to believe its a bug, because the window is definitely (buggily) responding to the hotkey despite it not existing anywhere. i have no other window managers installed

**Reply by 9mm** [reply]
Sry again, i just restarted computer and that seemed to resolve the quirkiness.... sweet!
